_datalad_buildsupport/formatters.py

```diff
@@ -75,7 +75,7 @@ def _mk_title(self, prog):
 
     def _mk_name(self, prog, desc):
         """
-        this method is in consitent with others ... it relies on
+        this method is in consistent with others ... it relies on
```

Comment:
is this supposed to say "inconsistent"?

Reply:
or "in consistence"?

Reply:
who knows... stems all the way to its roots in datalad core. We better unify/fix all this goodness in https://github.com/datalad/datalad-buildsupport/ and use it through out. But I have not done my attempt in https://github.com/datalad/datalad-buildsupport/pull/3 -- we would need to tune up for extensions vs not, and release as a proper package to not carry this damn copies all over....  overall I would not bother fixing this here.

Reply:
yeah -- probably was that:

```suggestion
        this method is inconsistent with others ... it relies on
```